Finmer.Game/Views/CharCreateBasic.xaml.cs

```diff
@@ -77,9 +92,11 @@ private void optGender_Checked(object sender, RoutedEventArgs e)
 
         private void ValidateForm()
         {
+            //add additional check to make sure radio button has been selected.
```

Comment:
Please remove this comment or rephrase it so it doesn't refer to only the gender radio buttons (something along the lines of "Ensure the user has provided valid values for all fields"). The comment as it is is more of a note about the Git commit than about the code, which may be confusing for future readers.